wiki/osu!stream/Song_List/en.md

```diff
@@ -1,210 +1,286 @@
-  - [@portal2news](https://twitter.com/portal2news)
+- [Souleye's website](http://magnuspalsson.com/)
 
-## VVVVVV Pack Vol. 1
+## SHIKI Pack Vol. 1
 
-Price: **USD$0.99**
+Price: **Free** / **USD$1.99** (pre-[2020 update](https://news.osustream.com/post/611013728007274496/osustream-2020-release))
 
-**Packlist:*
+This song pack features four songs by renowned BMS music producer SHIKI.
 
-- [Souleye - Positive Force](https://osu.ppy.sh/beatmapsets/12952) - [RandomJibberish](https://osu.ppy.sh/users/RandomJibberish) - ![2.5](img/dif5.png "2.5")
-- [Souleye - Pressure Cooker](https://osu.ppy.sh/beatmapsets/24271) - [Xgor](https://osu.ppy.sh/users/Xgor) - ![3](img/dif6.png "3")
-- Souleye - Potential for Anything - [Larto](https://osu.ppy.sh/users/12328) - ![3.5](img/dif7.png "3.5")
-- Souleye - Passion for Exploring - [mm201](https://osu.ppy.sh/users/mm201) - ![1.5](img/dif3.png "1.5")
+| Song | Mapper | Difficulty Rating |
+| :-- | :-- | :-- |
+| SHIKI - Lapis | Team STREAM (![][flag_GB] [RandomJibberish](https://osu.ppy.sh/users/157879) and ![][flag_FR] [Kurai](https://osu.ppy.sh/users/98661)) | ![5/10](img/dif5.png "5/10") |
+| SHIKI - Pure Ruby | ![][flag_US] [Lybydose](https://osu.ppy.sh/users/64501) | ![9/10](img/dif9.png "9/10") |
+| SHIKI - BABYLON | ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661) | ![8/10](img/dif8.png "8/10") |
+| SHIKI - Jade Star | ![][flag_KR] [Nakagawa-Kanon](https://osu.ppy.sh/users/87065) | ![7/10](img/dif7.png "7/10") |
 
-**Source:*
+See also:
 
-- [Souleye's website](http://magnuspalsson.com/)
+- [SHIKI's website](http://shiki2.sakura.ne.jp/)
 
-## SHIKI Pack Vol. 1
+## Unreleased Tracks Vol. 1
 
-Price: **USD$1.99**
+Price: **Free** / **USD$1.99** (pre-[v1.1 update](https://news.osustream.com/post/10624373514/osustream-v11-release))
 
-**Packlist:*
+This song pack features four songs/arrangements of original Touhou Project compositions by various musicians from across the world. 
 
-- [SHIKI - Lapis](https://osu.ppy.sh/beatmapsets/19230 ) - Team STREAM - ![2.5](img/dif5.png "2.5")
-- [SHIKI - Pure Ruby](https://osu.ppy.sh/beatmaps/82604) - [Lybydose](https://osu.ppy.sh/users/Lybydose) - ![4.5](img/dif9.png "4.5")
-- [SHIKI - BABYLON](https://osu.ppy.sh/beatmapsets/31373) - [Xgor](https://osu.ppy.sh/users/Xgor) - ![4](img/dif8.png "4")
-- [SHIKI - Jade Star](https://osu.ppy.sh/beatmapsets/19221) - [Nakagawa-Kanon](https://osu.ppy.sh/users/Nakagawa-Kanon) - ![3.5](img/dif7.png "3.5")
+Initially, this song pack was included in the base game's launch with a price tag of USD$1.99. However, due to legal concerns over the distribution of assets related to Touhou Project in the App Store as a paid content, this song pack was later pulled in the [v1.1 update](https://news.osustream.com/post/10624373514/osustream-v11-release). This song pack has consequently been re-released as a free content in [the 2020 update](https://news.osustream.com/post/611013728007274496/osustream-2020-release).
```

Comment:
It was never published. The rules regarding touhou distribution on iOS changed halfway through the development process, after we finished mapping and licensing all the music. I contacted ZUN as goodwill and got a strict "don't use" so we had to refund / retract all contracts and find new music.

Reply:
I guess it was released very briefly, according to the linked article! Has been a long time..

Reply:
yup! i don't have an iOS device myself but i also remember downloading the early (v1.0x) releases of the app back then and seeing the Touhou songs in the Store listing when i installed osu!stream and [made this score](http://osustream.com/p/ranking?v=score&sc=106189&d=1) on my friend's device ;~;